liste.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<!--
+  SPDX-License-Identifier: AGPL-3.0-only
+  SPDX-FileCopyrightText: 2025 Tim Beckmann <beckmann.tim@fh-swf.de>
+-->
+
+<html lang="de">
+
+<head>
+    <load src="partials/meta.html" />
+    <title>List | EinkaufsApp</title>
+    <link href="src/list.css" type="text/css" rel="stylesheet" />
+    <script type="module" src="/src/list.ts"></script>
+</head>
+
+<body>
+    <load src="partials/standardheader.html" pageTitle="Anmeldung" />
+    <main>
+        <div id="listInfo">
+            <input type="date" id="deadline" disabled required>
+            <input type="text" id="listTitel" disabled required>
+            <br/>
+            <textarea id="listDescription" rows="4" cols="50" disabled required></textarea>
+            <br />
+            <button type="button" id="editDescriptionButton">&#9998;</button>
+        </div>
```

Comment:
the button to delete the list is missing. i know there's one on the dashboard already, but having both options just seems logical.